main.py

```diff
@@ -11,15 +11,21 @@ def hello():
 def get_bodies():
     try:
         bodies = CelestialBodies.query.all()
-        return jsonify({'data': [e.serialize() for e in bodies]})
+        if not bodies:
+          return jsonify({'errors': 'No celestial bodies found.'}), 404
+        else:
+          return jsonify({'data': [e.serialize() for e in bodies]})       
```

Comment:
I added some minor sad paths to make it easier for anyone hitting the API to know what the error is and hopefully communicate what is broken to us more easily.